code/controllers/subsystem/timer.dm

```diff
@@ -43,26 +43,25 @@ SUBSYSTEM_DEF(timer)
 		WARNING(msg)
 		if(bucket_auto_reset)
 			bucket_resolution = 0
-		
-		log_world("Active timers at tick [world.time]:")
+
+		log_world("Timer bucket reset. world.time: [world.time], head_offset: [head_offset], practical_offset: [practical_offset]")
+		for (var/i in 1 to length(bucket_list))
+			var/datum/timedevent/bucket_head = bucket_list[i]
+			if (!bucket_head)
+				continue
+				
+			log_world("Active timers at index [i]:")
+			
+			var/datum/timedevent/bucket_node = bucket_head
+			var/anti-loop-check = 1000
```

Comment:
web edit so this isn't a define. this debug info will likely get removed once this bug is fixed, so i'm not worried, but i thought i'd preempt @RemieRichards